tests/detect-bytetest-03/test.rules

```diff
@@ -0,0 +1,3 @@
+alert dns any any -> any any (msg:"Byte test against first byte"; byte_test:1,=,0x07,0,bitmask 0xF8; sid:1;)
```

Comment:
`bitmask` was not implemented in `5.0.x`, please add a min-version check in this test for 6.